app/models/url.rb

```diff
@@ -0,0 +1,50 @@
+class Url < ActiveRecord::Base
+  has_many :payloads
+
+  validates :root_url, presence: true
+  validates :path, presence: true
+
+  def max_response_time
+    payloads.maximum(:responded_in)
+  end
+
+  def min_response_time
+    payloads.minimum(:responded_in)
+  end
+
+  def response_times_desc
+    payloads.order(responded_in: :desc).map do |e|
```

Comment:
same here - what is `e`? i should be able to tell what this is with a glance instead of having to digest the entire method